### PR TITLE
MacOS 10.15 support

### PR DESCRIPTION
**Pull request by chrisandreae**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -13,14 +13,38 @@ if ! [ -d "$EIKAIWA_BASEDIR/eikaiwa_content" ]; then
...
+        echo "Added /nix and /run to synthetic.conf. You must now reboot and re-run this script."
+        exit 0
+    fi
+
+    # If 10.15 and nothing mounted on /nix
+    if ! mount | grep -q 'on /nix'; then
+        echo "Creating and mounting /nix filesystem"
+        sudo diskutil apfs addVolume disk1 'Case-sensitive APFS' Nix -mountpoint /nix
+        sudo diskutil enableOwnership /nix
+        echo 'LABEL=Nix /nix apfs rw' | sudo tee -a /etc/fstab >/dev/null
```
Comment (thefloweringash) [inline install.sh:47]: Is this line necessary or are apfs volumes self-assembling?

**Review by thefloweringash — APPROVED** [review]

File: install.sh [hunk 2]
```diff
@@ -13,14 +13,38 @@ if ! [ -d "$EIKAIWA_BASEDIR/eikaiwa_content" ]; then
...
 # Authenticate to sudo: nix installers are going to need it, but we're detaching
 # them from the terminal
 echo "Obtaining sudo:"
 sudo echo "Obtained!"
 
+# If MacOS 10.15
+darwin_version="$(uname -r)"
+if [ "${darwin_version%%.*}" -ge 19 ]; then
+    if ! [ -d "/nix" ]; then
+        (echo 'nix'; echo 'run	private/var/run') | sudo tee -a /etc/synthetic.conf >/dev/null
```
Comment (thefloweringash) [inline install.sh]: Consider `echo -e "\t"` over literal tab

**Comment by kevingriffin** [comment]
https://www.apple.com/newsroom/2019/10/macos-catalina-is-available-today/

File: install.sh [hunk 3]
```diff
@@ -13,14 +13,38 @@ if ! [ -d "$EIKAIWA_BASEDIR/eikaiwa_content" ]; then
...
+        echo "Added /nix and /run to synthetic.conf. You must now reboot and re-run this script."
+        exit 0
+    fi
+
+    # If 10.15 and nothing mounted on /nix
+    if ! mount | grep -q 'on /nix'; then
```
Comment (szymonrw) [inline install.sh]: Will this work on non-English locales?

File: install.sh (hunk 1 above)
Comment (chrisandreae) [reply install.sh:47]: I was under the impression it was; if it's not we can remove it.

File: install.sh (hunk 2 above)
Comment (chrisandreae) [reply install.sh]: fixed

File: install.sh (hunk 3 above)
Comment (chrisandreae) [reply install.sh]: took your advice and added `LANG=en_US` (ew!)

**Pull request merged**